commit cc79cad299395f11a5df9da0d6a8991a118f2b81
Author: Greg Kroah-Hartman <gregkh@suse.de>
Date:   Tue Oct 5 09:51:42 2010 -0700

    Staging: brcm80211: typedefs.h: remove version.h inclusion
    
    It isn't needed anymore.
    
    Cc: Brett Rudley <brudley@broadcom.com>
    Cc: Henry Ptasinski <henryp@broadcom.com>
    Cc: Nohee Ko <noheek@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/include/typedefs.h
+++ b/drivers/staging/brcm80211/include/typedefs.h
@@ -31,7 +31,6 @@ typedef unsigned long long int uintptr;
 #define TYPEDEF_UINT
 #define TYPEDEF_USHORT
 #define TYPEDEF_ULONG
-#include <linux/version.h>
 #include <linux/types.h>
 
 /*
